Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.py

```diff
@@ -1537,6 +1538,14 @@ def search_file_command(file):
     return command_results
 
 
+def get_tags_for_generic_conext(tags: dict):
```

Comment:
It looks like 'tags' can be None. Please take care of this case (you can simply return None and then the creation of the File will just ignore this field).

Also add Optional[dict] to the method's type annotation - 
![image](https://user-images.githubusercontent.com/50294648/107876221-59455880-6ecd-11eb-862c-1384178c5135.png)

Reply:
Done